RandomError1.py

```diff
@@ -0,0 +1,226 @@
+        if ASCII is 'G':
+            Pscore+=.00016
+        if ASCII is 'H':
+            Pscore+=.00013
+        if ASCII is 'I':
+            Pscore+=.00010
+        if ASCII is 'J':
+            Pscore+=.00008
+    Pscorelist.append(Pscore)
+    return Pscorelist
+
+def phredscore(fastq_fh):
+    Pscorelist=[]
+    for line in fastq_fh:
+        if 'B' in line:
+            Pscorelist=phredvalues_version8(line)
+            return Pscorelist
+
+
+
+
+
+
+def errorproducer(fastq_fh):
+    errorcount=0
+    totaltotalbase=0
+    Pscoreplacemark=0
+    dankhold=0
+    Pscorelist=[]
+    for line in fastq_fh:
+        if dankhold==0:
+            Pscorelist=phredscore(fastq_fh)
+        dankhold=1
+        totalbase=0
+        placemark=0
+        newstring=""
+        line=line.rstrip()
+        #getting rid of \'s
+        if 'B' in line:
+            continue
+        if '@' in line:
+            continue
+        if '}' in line:
+            continue
+        if '+' in line:
+            continue
+        if '-' in line:
+            continue
+        if ';' in line:
+            continue
+        #really sloppily discounting any lines apart from those with basepairs
+        totalbase=len(line)
+        totaltotalbase+=totalbase
+        Pscoreplacemark=1
+        print Pscorelist
+        for basepair in line:
+            placemark+=1
+            errorchance=Pscorelist[Pscoreplacemark-1]
+            int(errorchance)
+            genedice=random.random()
+            #checks place in line then produces a random number based on it
+            if genedice > errorchance:
+                newstring+=basepair
+                continue
+                #if the number generated is higher than the produced error chance, the base pair is added to a new line
+            else:
+                errorcount+=1
+                if basepair==('G'):
+                    if genedice < errorchance/3:
+                        newstring+='A'
+                        continue
+                    if genedice < 2*errorchance/3:
+                       newstring+='C'
+                       continue
+                    if genedice <= errorchance:
+                        newstring+='T'
+                        continue
+                if basepair==('C'):
+                    if genedice < errorchance/3:
+                        newstring+='G'
+                        continue
+                    if genedice < 2*errorchance/3:
+                       newstring+='A'
+                       continue
+                    if genedice <= errorchance:
+                        newstring+='T'
+                        continue
+                if basepair==('A'):
+                    if genedice < errorchance/3:
+                        newstring+='G'
+                        continue
+                    if genedice < 2*errorchance/3:
+                        newstring+='C'
+                        continue
+                    if genedice <= errorchance:
```

Comment:
It's not divided by 3 so it compares the genedice to the whole number as opposed ti 1/3 and 2/3 of the number such as in the previous two. The essentially sequence goes through and checks which range, between 0 and the 1/3, 2/3 then 3/3 the rolled number fits into (in that order).